hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1284,7 +1284,7 @@ public static com.hedera.hapi.node.base.Key asPbjKey(@NonNull final JKey jKey) {
         requireNonNull(jKey);
         try {
             return toPbj(JKey.mapJKey(jKey));
-        } catch (DecoderException e) {
+        } catch (InvalidKeyException e) {
             throw new RuntimeException(e);
```

Comment:
```suggestion
            throw new IllegalStateExcepion("Can not convert key",e);
```

Reply:
I'm not cleaning up the large base of not-well-written code in this PR.  Only changing it to not use DecoderException.  All of these items are pre-existing.
I don't know the cascade for changing the translated exception type, in particular, here, so I'm not going to risk changing it.